app/lending/(2-guides)/understanding-liquidations/page.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: Soft vs Hard Liquidations
+description: The difference between soft and hard liquidation in Chainflip
+---
+
+## What is a soft liquidation?
+
+Soft liquidation is the first response when your LTV crosses the liquidation threshold.
+
+- Triggered automatically
+- Only a portion of collateral is sold
+- Uses tight live price protection (0.5%)
+- Executed through DCA-style swaps
+- Abortable if LTV improves during the process
```

Comment:
Since we say that it is triggered automatically, might as well say that it may be aborted automatically too.
```suggestion
- Automatically aborts if LTV improves during the process
```